### PR TITLE
Added Content Management Api documentation for the Heartcore client library

### PR DESCRIPTION
**Pull request by devantler**
This PR provides documentation on how to use the `ContentManagementService` to manage content and media. It has been created to solve customer issues regarding File Upload and Image Cropper upload requiring raw property values for the specific property editor.

### CODE REVIEW

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+    ```csharp
+    var content = await _contentManagementService.Content.GetById(yourGuidObject);
+    //or
+    var content = await _contentManagementService.Content.GetById(Guid.Parse("your-guid-as-a-string"));
+    ```
+
+3. `GetChildren(Guid id)` - Gets all content that is a child of a specific content item matching a GUID.
+
+    To use this method, call the method on the `ContentManagementService` instance and pass in the GUID of the content item you want to get the children of:
```
Comment (sofietoft) [inline Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md:87]:
```suggestion
    To use this method, call the method on the `ContentManagementService` instance and pass the GUID of the content item to get the children:
```

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+    ```csharp
+    var deletedContent = await _contentManagementService.Content.Delete(yourGuidObject);
+    //or
+    var deletedContent = await _contentManagementService.Content.Delete(Guid.Parse("your-guid-as-a-string"));
+    ```
+
+## Media
+
+First, we will look at how to work with media. The `ContentManagementService` allows you to Get, Create, Update and Delete media.
```
Comment (sofietoft) [inline Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md:187]: Content was "first", so I guess Media also can't be "first"? 😄 

I think, let's take out the "First"s all together. I don't people will be reading an article like this from top to bottom anyways, right?
What do you think?

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+    ```csharp
+    var media = await _contentManagementService.Media.GetById(yourGuidObject);
+    //or
+    var media = await _contentManagementService.Media.GetById(Guid.Parse("your-guid-as-a-string"));
+    ```
+
+3. `GetChildren(Guid id)` - Gets all media that is a child of a specific media item matching a GUID.
+
+    To use this method, call the method on the `ContentManagementService` instance and pass in the GUID of the media item you want to get the children of:
```
Comment (sofietoft) [inline Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md:236]:
```suggestion
    To use this method, call the method on the `ContentManagementService` instance and pass in the GUID of the media item to get the children:
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @devantler ! Looks great 😄 💪 

I've added a couple of suggestions and comments.
Let me know if something doesn't make sense.

File: Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md (hunk 2 above)
Comment (devantler) [reply Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md]: I agree. I reworded this to align with your thoughts. What do you think? :smiley:

**Comment by sofietoft** [comment]
Excellent @devantler ! I'm happy with the changes.

I'll make sure to merge it in.

Also, I'll be migrating this to the new platform (probably on Friday) - I might want to pick your brain about making a couple of minor adjustments to the structure of the article. I'll keep you posted 🙌

**Pull request merged**